lib/launcher.ts

```diff
@@ -296,9 +300,13 @@ let initFn = function(configFile: string, additionalConfig: Config) {
               }
               return cleanUpAndExit(exitCode);
             })
-            .done();
+            .catch(err => setTimeout(() => {
```

Comment:
There are a few spots with 

```javascript
      // ...
    }).done();
  }).done();
```
I think that there are a few talks that would be beneficial, in terms of scope of remapping some of the flow of the application.

I will, though, move my `tap` and the like, out to the `util` file as higher-order functions that can be configured in the scope they expect to operate within.